tests/phpunit/tests/blocks/wpBlockType.php

```diff
@@ -84,6 +84,38 @@ public function test_set_props() {
 		$this->assertSame( $args['foo'], $block_type->foo );
 	}
 
+	public function test_core_attributes() {
```

Comment:
```suggestion
	/*
	 * @ticket 55567
	 * @covers WP_Block_Type::set_props
	 */
	public function test_core_attributes() {
```

:1234: Applies to `test_core_attributes_matches_custom` too